app/validators/questionnaire_validator.py

```diff
@@ -87,10 +87,11 @@ def validate_smart_quotes(self):
                 values_to_check = schema_text.values()
 
             for schema_text in values_to_check:
-                found = quote_regex.search(schema_text)
-
-                if found:
-                    self.add_error(
-                        error_messages.DUMB_QUOTES_FOUND,
-                        pointer=translatable_item.pointer,
-                    )
+                if schema_text is not None:
+                    found = quote_regex.search(schema_text)
+
+                    if found:
```

Comment:
I think this can be simplified to:
```suggestion
                if schema_text and quote_regex.search(schema_text):
```